packages/mdc-notched-outline/mdc-notched-outline.scss

```diff
@@ -29,11 +29,8 @@
 // Notched Outline is intended for use by multiple components, but its styles should only be emitted once when bundled
 @include mdc-base-emit-once("mdc-notched-outline") {
   .mdc-notched-outline {
-    position: absolute;
-    top: 0;
-    left: 0;
-    width: calc(100% - 1px);
-    height: calc(100% - 2px);
+    @include mdc-notched-outline-base_;
```

Comment:
Rather than put this in a mixin, should we separate these styles and put them under `.mdc-notched-outline, .mdc-notched-outline__idle`?